samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/ItemContentsBean.java

```diff
@@ -1390,6 +1390,24 @@ public String getAnswerKeyTF() {
 	return answerKey;
   }
 
+  public String getAnswerKeyCalcQuestion() {
+	String answerKey = "";
+	if(itemData!=null){
+		answerKey = itemData.getAnswerKey();
+		String keys[] = answerKey.split(",");
+		answerKey = "";
```

Comment:
Hello and thanks for your comment,
The code looks a little bit odd because I have reused the answerKey var in order to get itemData.getAnswerKey() and then, once splitted answerKey is set again to "".
I have changed the PR adding a new var answerKeyToSplit to make it more readable and I have added a new checking to prevent null values